main/views.py

```diff
@@ -167,5 +167,5 @@ def api_match_detail(request, match_id):
 
 @require_POST
 def api_remove_invalid_matches(request):
-    remove_invalid_matches()
-    return HttpResponse("Invalid matches removed")
+    response = remove_invalid_matches()
+    return HttpResponse("The list of invalid ids removed is: " + str(response))
```

Comment:
I just recommend sending back the json of the removed matches/match ids. Noone is interested in text.
🔠

Reply:
So does this mean I should save the raw json to a variable of each match, then remove the match, then return the json?

Reply:
Let's talk about this in person today.